CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - server: add support for `_inc` on `real`, `double`, `numeric` and `money` (fix #3573)
 - server: support special characters in JSON path query argument with bracket `[]` notation, e.g `obj['Hello World!']` (#3890) (#4482)
 - server: add graphql-engine support for timestamps without timezones (fix #1217)
+- server: support inserting unquoted bigint, and generate an integer overflow error on insert (fix #576) (fix #4368)
```

Comment:
@abooij Do you want to say something like `...., and generate an error if value overflows the bounds of the integer type`

Reply:
Yes, that is what I intended to express. I'll rephrase it following your suggestion :)